src/game_party.h

```diff
@@ -306,6 +306,19 @@ class Game_Party : public Game_Party_Base {
 	 * Removes invalid actors and items from the party
 	 */
 	void RemoveInvalidData();
+
+	std::vector<int16_t> GetInflictedStates() const;
+
+	/**
+	* Applies damage to the game party based on their stats.
+	*
+	* @return Whether the actor suffered some damage.
+	*/
```

Comment:
You messed up the * again. This automatic formating is really annoying Visual Studio behaviour ^^'
Can be fixed as part of some other PR: